server/client-shared/src/main/scala/cool/graph/client/requestPipeline/FunctionExecutor.scala

```diff
@@ -191,10 +230,14 @@ class FunctionExecutor(implicit val inj: Injector) extends Injectable {
               FunctionExecutionResult(Vector.empty, parsed)
           }
 
-          def getResult(data: Any): FunctionDataValue = function match {
+          def getResult(data: Any): FunctionDataValue Or FunctionError = function match {
             case f: CustomQueryFunction    => parseResolverResponse(data, f.payloadType)
```

Comment:
Style: If one case exceeds one line, break all cases to newlines and have a blank line in between each case.

Reply:
done